index.js

```diff
@@ -1,65 +1,70 @@
-// Book Object
-const Book = {
-  title: '',
-  author: '',
-};
-// Books Collection
-let Books = [];
-if (localStorage.Books) {
-  Books = JSON.parse(localStorage.Books);
+class Books {/* eslint-disable-line max-classes-per-file */
+  constructor() {
+    this.array = [];
+  }
 }
 
-// populate local storage collection with user entries
+class Book {/* eslint-disable-line max-classes-per-file */
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
```

Comment:
- [ ] Kindly add to your class two methods the `add` and `remove` methods as asked by the requirements `Create class methods to add and remove books.`.